test/print-routes.test.js

```diff
@@ -19,20 +20,16 @@ test('should print routes', async t => {
 
   await fastify.close()
   t.ok(spy.called)
-  t.same(spy.args, [['debug', '└── / (GET)\n    / (HEAD)\n    / (POST)\n']])
+  t.same(spy.args, [['debug', '└── / (GET, HEAD, POST)\n']])
 })
 
 test('should print routes via cli', async t => {
```

Comment:
This test seemed to do the exact same thing as the prior test, so I re-wrote it to actually run through the CLI. Hope this is right.